packages/lodestar/src/network/gossip/utils.ts

```diff
@@ -29,7 +28,14 @@ export function getAttestationSubnet(attestation: Attestation): string {
   return String(attestation.data.index % ATTESTATION_SUBNET_COUNT);
 }
 
-export function deserializeGossipMessage<T>(msg: IGossipMessage, type: AnySSZType): T {
-  assert(msg.data.length <= GOSSIP_MAX_SIZE, `Message exceeds size limit of ${GOSSIP_MAX_SIZE} bytes`);
-  return deserialize(type, msg.data);
+export function isAttestationSubnetTopic(topic: string): boolean {
+  return AttestationSubnetRegExp.test(topic);
+}
+
+export function getSubnetFromAttestationSubnetTopic(topic: string): number {
```

Comment:
You should be able to capture the subnet with regex: https://javascript.info/regexp-groups